webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsTableSchema.js

```diff
@@ -40,38 +43,38 @@ export const columns = (controller, selectionController) => [
     property: 'contract_number',
     header: {
       label: __('Contract'),
-      formatters: [headerFormat],
+      formatters: [headerFormatter],
     },
     cell: {
-      formatters: [cellFormat],
+      formatters: [cellFormatter],
```

Comment:
Same thing like `headerFormatter`.

BTW, I don't think `formatters` is useless, I am using `formatters` to wrap `value` with a `<a>` - generating the URL from `{rowData}` or to display `value` as an action button.